index.html

```diff
@@ -0,0 +1,140 @@
+              <div class="desc">
+                <h2>Fast Shipping</h2>
+                <p>
+                  We make sure you recieve your trombone as soon as we have
+                  finished making it. We also provide free returns if you are not
+                  satisfied.
+                </p>
+              </div>
+            </div>
+            <div class="grid">
+              <div class="icon">
+                <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+              </div>
+              <div class="desc">
+                <h2>Quality Asurance</h2>
+                <p>
+                  For every purchase you make, we will ensure there are no damages
+                  or faults and we will check and test the pitch of your
+                  instrument.
+                </p>
+              </div>
+            </div>
+          </section>
+          <section id="how-it-works">
+            <iframe
+              id="video"
+              height="315"
+              src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+              frameborder="0"
+              allowfullscreen
+            ></iframe>
+          </section>
+          <section id="pricing">
+            <div class="product" id="tenor">
+              <div class="level">Tenor Trombone</div>
+              <h2>$600</h2>
+              <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="btn">Select</button>
+            </div>
+            <div class="product" id="bass">
+              <div class="level">Bass Trombone</div>
+              <h2>$900</h2>
+              <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="btn">Select</button>
+            </div>
+            <div class="product" id="valve">
+              <div class="level">Valve Trombone</div>
+              <h2>$1200</h2>
+              <ol>
+                <li>Plays similar to a Trumpet</li>
+                <li>Great for Jazz Bands</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="btn">Select</button>
+            </div>
+          </section>
+          <footer>
+            <ul>
+              <li><a href="#">Privacy</a></li>
+              <li><a href="#">Terms</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+            <span>Copyright 2016,Original Trombones</span>
+          </footer>
+        </div>
+      </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
```

Comment:
Don't need to add a js file now for this project. Please remove it.

Reply:
- [ ] Please be more polite while doing the review, and also when you are doing review kindly point the reason that why you asked the student to remove the js file.
### Example 👇
```
The student comments are clear and easy to understand. 

* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* This is a vague comment: *The README is not descriptive, why don't you make it better?*
```